include/meshlink/meshlink.h

```diff
@@ -304,10 +304,6 @@ typedef void (*meshlink_log_cb_t)(meshlink_handle_t *mesh, meshlink_log_level_t
  *  to hand the data over to the application's thread.
  *  The callback should also not block itself and return as quickly as possible.
  *
- *  The @a mesh @a parameter can either be a valid MeshLink handle, or NULL.
```

Comment:
Should this part of the comment be removed? Looking `meshlink.c` this looks fine to me.